tests/fixtures/pnpm-lock.yaml

```diff
@@ -478,3 +478,10 @@ packages:
     name: testing
     version: 1.0.0
     dev: false
+
+  file:projects/workspace_member:
+    resolution: {directory: projects/workspace_member, type: directory}
+    name: workspace_member
+    dependencies:
+      typescript: 5.2.2
```

Comment:
Since this entry would normally also be in the same lockfile, it might be worth adding it to this test lockfile...so it doesn't trip someone up (like it did me). It would be this block added:

```
  /typescript@5.2.2:
    resolution: {integrity: sha512-mI4WrpHsbCIcwT9cF4FZvr80QUeKvsUsUvKDoR+X/7XHQH98xYD8YHZg7ANtz2GtZt/CBq2QJ0thkGJMHfqc1w==}
    engines: {node: '>=14.17'}
    hasBin: true
    dev: false
```

If so, the number of packages parsed in the test would need to be bumped up by one.

Reply:
You'll find that many of our test lockfiles are written by hand and not 100% correct. But their purpose is to test our parser not necessarily to be correct lockfiles.

I've still fixed this just because you asked so nicely. :)